scripts/main.js

```diff
@@ -5,4 +5,5 @@ function setup(){
 
 function draw(){
     background("#333");
-}
\ No newline at end of file
+    circle(width/2, height/2, 20,20);
```

Comment:
circle function accepts only two parameters.
Here you should delete last parameter

Reply:
Thanks for pointing that

Reply:
Now I have fixed that